src/2982R0_std_quantity_as_a_numeric_type.md

```diff
@@ -2318,40 +2328,43 @@ This has significant advantages.  It preserves the _form_ of the code, making th
 than replacement with a function such as `is_gt_zero`.  It also reduces the number of new comparison APIs
 a user must learn: `Zero` handles them all.
 
-`Zero` has a few downsides though:
-
-1. CTAD can't deduce the quantity type as there is no unit provided. So if we try to refactor the
-   following:
-
-    ```cpp
-    quantity<si::metre / si::second> q1 = 0 * m / s;  // OK
-    quantity q2 = 0 * m / s;                          // OK
-    ```
-
-   only the first one will work and the second one will fail to compile:
+`Zero` has one downside: it will not work when passed across _generic quantity_ interfaces.  `Zero`'s value
+comes in situations where the surrounding context makes it unambiguous which quantity type it should
+construct.  While it converts to any _specific_ quantity type, it is not itself a quantity.  This could
+confuse users.
 
-    ```cpp
-    quantity<si::metre / si::second> q1 = ZERO;  // OK
-    quantity q2 = ZERO;                          // Compile-time error
-    ```
+This downside manifests in several different ways.  Here are some examples:
 
-2. While refactoring the [@MP-UNITS] code to try out this approach we found out also some other places
+1. While refactoring the [@MP-UNITS] code to try out this approach we found out a perfectly reasonable place
    where we could not replace numerical value `0` with `ZERO`:
 
    ```cpp
-   for (auto tt = 0 * ms; tt <= 50 * ms; ++tt) { /* ... */ }
+   msl_altitude alt = mean_sea_level + 0 * si::metre;  // OK
+   msl_altitude alt = mean_sea_level + ZERO;           // Compile-time error
```

Comment:
Maybe use different identifiers for each case (e.g. `alt1` and `alt2`)?

Reply:
SGTM!